dts/bindings/pinctrl/nordic-clock-enable.yaml

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+properties:
+  nordic,clock-enable:
```

Comment:
it would be better to rename it to `nordic,clockpin-enable` otherwise it might be confusing suggesting that some clocks are enabled. bit field in GPIO register is called `clockpin`.

Reply:
renamed, good idea.